server.js

```diff
@@ -246,6 +327,9 @@ app.get('/game/:id/player/:pid/bestmove', validate_gid, validate_pid,
         });
 
         return;
+    } else { // if ( end turn situation) 
```

Comment:
Unnecessary else. Also, `res.status(404)` is called twice which would result in an error.